EndlessClient/Dialogs/TextMultiInputDialog.cs

```diff
@@ -136,6 +153,8 @@ public TextMultiInputDialog(INativeGraphicsManager nativeGraphicsManager,
             DialogClosed += (_, _) => chatTextBoxActions.FocusChatTextBox();
 
             CenterInGameView();
+
+            DrawPosition += new Vector2(-160, 0);
```

Comment:
Test this in both resizable and non-resizable mode, I feel like this'll be broken in a fixed viewport

Reply:
Ended up overriding/rewriting `CenterInGameView` because the base class functionality doesn't really work with how background texture is handled in this dialog.